### PR TITLE
Add runbook for content-fluentd-stream

### PR DESCRIPTION
**Pull request by ekaneva**
# Add runbook for content-fluentd-stream

## What

Add a runbook to this kinesis stream to improve the general SOS score.

## Why

[JIRA ticket](https://financialtimes.atlassian.net/browse/UPPSF-1181)

## Anything, in particular, you'd like to highlight to reviewers

It is more special than than our regular service runbook. @Financial-Times/metadata-team and @Financial-Times/platform-health teams, please review it with care. I filled it out to the best of my knowledge. Thank you!

## Scope and particulars of this PR (Please tick all that apply)

- [x] Tech hygiene (dependency updating & other tech debt)
- [ ] Bug fix
- [ ] Feature
- [x] Documentation
- [ ] Breaking change
- [ ] Minor change (e.g. fixing a typo, adding config)

___
This Pull Request follows the rules described in our [Pull Requests Guide](https://github.com/Financial-Times/upp-docs/tree/master/guides/pr-guide)

### CODE REVIEW

**Review by mihaylovmihail — APPROVED** [review]

File: runbooks/content-fluentd-kinesis-runbook.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+# UPP - Content Fluentd Kinesis Stream Provisioner
```
[the remark this answers is not in the document]
Comment (ekaneva) [reply runbooks/content-fluentd-kinesis-runbook.md:1]: You are absolutely right. I don't know why at some point made it a runbook for the provisioner, Looking at it now, it does not make any sense.. Will address all related issues accordingly.

File: runbooks/content-fluentd-kinesis-runbook.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+NotApplicable
+
+## Data Recovery Process Type
+
+NotApplicable
+
+## Data Recovery Details
+
+Does not store data, so it does not require any data recovery steps.
```
[the remark this answers is not in the document]
Comment (ekaneva) [reply runbooks/content-fluentd-kinesis-runbook.md:79]: Then what should I put in `## Data Recovery Process Type` what is the process of restoring its data if something goes wrong? The stream is in AWS, which means they guarantee for its data, is that right? Do we have any extra process around it?

[the remark this answers is not in the document]
Comment (MiroslavGatsanoga) [reply runbooks/content-fluentd-kinesis-runbook.md:79]: Yes, AWS guarantee the 24h retention, but after the data is lost. As far as I know we don't do anything extra regarding the Kinesis stream.

[the remark this answers is not in the document]
Comment (ekaneva) [reply runbooks/content-fluentd-kinesis-runbook.md]: Great, thanks for confirming, Miro. :)

**Review by MiroslavGatsanoga — APPROVED** [review]

**Review by mihaylovmihail — APPROVED** [review]

**Pull request merged**